src/main/app/src/app/zaken/zaak-create/zaak-create.component.ts

```diff
@@ -51,6 +52,7 @@ export class ZaakCreateComponent implements OnInit {
     private initiatorField: InputFormField;
     private locatieField: InputFormField;
     private medewerkerGroepFormField: MedewerkerGroepFormField;
+    private vertrouwelijkheidaanduidingField: AbstractChoicesFormField;
```

Comment:
Gebruik hier gewoon de SelectFormField i.p.v. de AbstractChoicesFormField. Dan is meteen duidelijk wat het is.

Reply:
Opgelost in opvolgende commit